app.js

```diff
@@ -0,0 +1,6 @@
+// The date and time is display to check connectivity with the html file.
+var dateToday = new Date();
+var date = dateToday.getDate() + "/" + (dateToday.getMonth()+1) + "/" + dateToday.getFullYear();
+var time = dateToday.getHours() + ":" + dateToday.getMinutes(); 
+// TODO -- add cero to minutes less than 10.
```

Comment:
there are another useful method available: `toLocaleTimeString` - which gives you the time value. [Documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString)
```js
var time = dateToday.toLocaleTimeString('en-US', { hour: '2-digit', minute: '2-digit' });
```
if it doens't make sense then you can skip it) I just want to point to the already build in helper's available.

Reply:
thanks, the documentation helped a lot, I'll change it too